python/ray/tune/analysis/experiment_analysis.py

```diff
@@ -757,14 +767,28 @@ def runner_data(self) -> Dict:
 
     def _get_trial_paths(self) -> List[str]:
         if self.trials:
-            _trial_paths = [t.logdir for t in self.trials]
+            # Get the relative paths from the trials to allow
+            # for changes in the self._local_base_dir.
+            _trial_paths = [
+                str(self._local_base_dir.joinpath(t.relative_logdir))
+                if t.relative_logdir
+                else t.logdir
+                for t in self.trials
+            ]
         else:
             logger.info(
                 "No `self.trials`. Drawing logdirs from checkpoint "
                 "file. This may result in some information that is "
                 "out of sync, as checkpointing is periodic."
             )
-            _trial_paths = [checkpoint["logdir"] for checkpoint in self._checkpoints]
+
+            _trial_paths = [
+                str(self._local_base_dir.joinpath(checkpoint["relative_logdir"]))
+                if checkpoint["relative_logdir"]
```

Comment:
```suggestion
                if checkpoint.get("relative_logdir")
```

Reply:
this is not going to change anything without a default value for get, right?

Reply:
Compare

```
>>> bool({"test":1}["not_found"])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
KeyError: 'not_found'
>>> bool({"test":1}.get("not_found"))
False
```

Reply:
Ah, good to know!

Reply:
Done